moveit/CHANGELOG.rst

```diff
@@ -2,6 +2,13 @@
 Changelog for package moveit
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+2.7.0 (2023-01-28)
+------------------
+* Fix BSD license in package.xml (`#1796 <https://github.com/JafarAbdi/moveit2/issues/1796>`_)
```

Comment:
```suggestion
* Fix BSD license in package.xml (`#1796 <https://github.com/ros-planning/moveit2/issues/1796>`_)
```

The PRs are linked to your branch instead of ros-planning

Reply:
Nice catch, catkin_generate_changelogs uses the origin remote's URL, I'll fix that